assets/src/blocks/Gallery/GalleryScript.js

```diff
@@ -1,10 +1,4 @@
 import { GalleryFrontend } from './GalleryFrontend';
+import { hydrateBlock } from '../../functions/hydrateBlock';
 
-document.addEventListener( 'DOMContentLoaded', () => {
-  const galleryBlocks = [...document.querySelectorAll('[data-render="planet4-blocks/gallery"]')];
-
-  galleryBlocks.forEach(blockNode => {
-    const attributes = JSON.parse( blockNode.dataset.attributes );
-    wp.element.render( <GalleryFrontend { ...attributes.attributes } />, blockNode );
-  });
-});
+hydrateBlock('planet4-blocks/gallery', GalleryFrontend, { renderLightbox: true });
```

Comment:
This is a very important change since the [frontendRendered](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/932/files#diff-73186c43f7b5ced8d9cc549a30d7a72ed30ff6fcace9c6cb2ae2236a5b9d7276L47) method is deprecated.